2-mandatory/4-stocks.js

```diff
@@ -34,9 +34,18 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  let averagePrice = [];
```

Comment:
This implementation looks good to me. One minor point would be the naming of variables. Normally if we're keeping track of multiple values (like in an array), we would use "stockPrices" instead "stockPrice".